app/react/Library/components/MapView.js

```diff
@@ -69,7 +69,7 @@ export class MapView extends Component {
             <b>{markers.get('totalRows')}</b> {t('System', 'documents')}
           </span>
         </div>
-        <Markers entities={markers.get('rows')}>
+        <Markers entities={markers.get('rows')} storeKey={storeKey}>
```

Comment:
Why does the Markers component need the storeKey property?

Reply:
It actually does not need storeKey, I forgot to remove when I realized entities are passed as a property. Removing...

Reply:
Removed